CONTRIBUTING.md

```diff
@@ -125,14 +131,12 @@ A PR is considered to be **ready to merge** when:
 * Trivial change (typo, cosmetic, doc, etc.) doesn't have to wait for one day.
 * Urgent fix can take exception as long as it has been actively communicated.
 
-Any Maintainer can merge the PR once it is **ready to merge**. Note, that some
-PR may not be merged immediately if repo is being in process of a major release
-and the new feature doesn't fit it.
+Any Maintainer can merge the PR once it is **ready to merge**.
```

Comment:
Note, that some
PR may not be merged immediately if repo is being in process of a major release
and the new feature doesn't fit it.


^ please keep this language.

Reply:
I have added it back.